src/main/java/com/ss/lms/controller/BorrowerController.java

```diff
@@ -0,0 +1,54 @@
+package com.ss.lms.controller;
+
+import java.sql.SQLException;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ss.lms.entity.Author;
+import com.ss.lms.entity.Book;
+import com.ss.lms.entity.Borrower;
+import com.ss.lms.entity.Branch;
+import com.ss.lms.entity.Loans;
+import com.ss.lms.service.BorrowerService;
+
+@RestController
+public class BorrowerController {
+	@Autowired
+	BorrowerService borrowerService;
+
+	@RequestMapping(path = "/api/books/checkout", method = RequestMethod.POST, headers = {
+			"Accept=application/json,application/xml" }, produces = { "application/json", "application/xml" })
```

Comment:
Im confused about whats happening in the header.  Are you making it only accept request that say application/json or app/xml in the header??

Or is it whenever a json file is sent the info saying the file is json is added to the header automatically?